### ISSUE TITLE
How to get the target from onClick event?

### ISSUE THREAD

**Issue by michaelsync** [JustFly1984/react-google-maps-api#95]
Hi, As of now, the onClick event is google.maps.MouseEvent. `onClick?: (e: google.maps.MouseEvent) => void;`. How do I get the market object in that click event? 

Lets say I have a code as below. I created the list of marker dynamically from the list. 

````
     list.forEach((tx:Cab) => }
        markers.push(
          <Marker
            key={tx.CarNumber}
            position={{ lat: tx.Latitude, lng: tx.Longitude }}
            label={carNumberAndDirection}
            onClick={(e:any) => {              
              console.log((e.target as Marker));
              
            }}
          />
  )
````

- OS: Windows
- "@react-google-maps/api": "1.2.1",
- Node: v10.15.1
- "react": "^16.8.4",
- "react-bootstrap": "^1.0.0-beta.6",
- "react-dom": "^16.8.4",
- "react-router": "^5.0.0",
- "react-router-dom": "^5.0.0",

**Comment by uriklar** [comment]
@michaelsync You currently can't get the instance from an onClick event. Can you please explain why you need the marker on click? Maybe I can help you find the correct way of doing what you're trying to do

**Comment by michaelsync** [comment]
@uriklar The reason is that I need to show the InfoWindows when a user clicks on the marker. I have the sample code in this post https://github.com/JustFly1984/react-google-maps-api/issues/94 ..  Now, the anchor property of InfoWindows is the required property so I guess I need to pass the instance of Marker to the anchor property.

**Comment by uriklar** [comment]
I'm looking at the code in #94 but I don't understand why you'd need the Marker instance on click. You've already enclosed it in the onClick callback. Your code should be working. Your only problem right now is the typescript error. 
Anchor is not required, that was a mistake that will be fixed in 1.2.3. You can pass the position property and that's good enough (though passing the marker as an anchor should work, but in your case the position property is more fitting)

**Comment by michaelsync** [comment]
@uriklar Thanks. Do you think you can release 1.2.3 with fixes soon? LIke Saturday. I've spent almost 1 and half day to debug the issue with 1.1.1 and then release it is working with 1.2.2 but not working with typescript. I tested 1.2.0 and 1.2.1.
